tests/Mutator/Removal/FunctionCallRemovalTest.php

```diff
@@ -82,6 +82,17 @@ public function provideMutationCases(): \Generator
 
 $this->foo();
 $a = 3;
+PHP
+        ];
+
+        yield 'It does not remove an assert() call' => [
```

Comment:
Please add a test where the function name is a variable or a string to make sure it doesn't crash there.

The `PregQuote` test is an example of how to do that.